monasca_rsyslog/client.py

```diff
@@ -46,13 +48,22 @@ def __init__(self):
             user_agent='rsyslog-monasca'
         )
         self._url = cfg.CONF.api.url
+        self.buffer = {}
+        self.interval = interval
+        self.start_time = time.time()
```

Comment:
Would we be able to flush the buffer if it exceeds some maximum size as set in the config file?

If it's simpler, we could just define a maximum number of messages which can be held before a flush.